pyqt-apps/siriushla/sirius_application.py

```diff
@@ -10,6 +10,8 @@
 
 from .util import get_window_id, set_style
 
+# Set the process umask so we don't have problems with logfile permissions
+umask = os.umask(0)
```

Comment:
Note that this function returns the previous umask, so I think it could just be `os.umask(0)`, without the assignment.

Reply:
done!